lidarTransX: -0.115
lidarTransY: 0
lidarTransZ: -0.325
ignoreMinDistSQ: 0.04
ignoreMaxDistSQ: 16.00
lidarYaw: 0.0
lidarRoll: 0.0
lidarPitch: 37
detectMinHeight: 0.07
detectMaxHeight: 0.5
camWatchAngleDeg: 75
addHPPerPoint: 3
hpMax: 30
minusHPPerFrame: 1
hpConsiderLive: 7
hpUnknownTempObstacle: -2
hpFree: 0
hpObstacle: -1
dvShrink: 3
minThroughRadius: 0.4
bestThroughRadius: 1